{{cookiecutter.plugin_repo}}/docs/installation.rst

```diff
@@ -6,7 +6,7 @@ Installation
 
   .. code-block:: bash
 
-    git clone https://github.com/{{ cookiecutter.organization_name }}./{{ cookiecutter.plugin_repo }}.git
+    git clone https://github.com/{{ cookiecutter.organization_name.lower().replace(' ', '') }}/{{ cookiecutter.plugin_repo }}.git
```

Comment:
It is better to user `cookiecutter.github_username` instead of `cookiecutter.organization_name`.